src/oci.rs

```diff
@@ -0,0 +1,226 @@
+use std::ffi::CString;
+
+extern crate libloading as lib;
+
+fn lib_file() -> String {
```

Comment:
This can just be `const lib_file: &str` instead of a function

Reply:
I think it would be better to use static linking here.